File: Travel managment system - C#/projekt/IBiuro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace projekt
{
    /// <summary>
    /// Interfejs IBiuro przechowujący metody dodawania, usuwania, wyszukiwania oraz sprawdzania czy dany lot jest już w biurze
    /// </summary>
    interface IBiuro
    {
        void DodajLot(Podroze t);
        void UsunLot(Podroze t);
        void UsunLot(string PESEL);
        bool CzyPodrozJestWBiurzePodrozy(Podroze podroz);
        List<Podroze> WyszukajPodroze(string wylot);
    }
}

File: Travel managment system - JAVA/src/com/company/IBiuro.java
package com.company;

import java.util.ArrayList;

/**
 * Interfejs IBiuro przechowujący metody dodawania, usuwania, wyszukiwania oraz sprawdzania czy dany lot jest już w biurze.
 */
public interface IBiuro {
    void DodajLot(Podroz t);
    void UsunLot(Podroz t);
    void UsunLot(String PESEL);
    boolean CzyPodrozJestWBiurzePodrozy(Podroz podroz);
    ArrayList<Podroz> WyszukajPodroze(String wylot);
}
